source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -324,6 +312,72 @@ export default class HardwareWalletsStore extends Store {
       });
     }
   };
+
+  waitForLedgerDevices = async () => {
```

Comment:
To make it a bit easier to understand the code for people not familiar with this area, I would suggest making the function a bit more descriptive - how about:
```suggestion
  waitForLedgerDevicesToConnect = async () => {
```

Reply:
(if you like this approach you'd need to rename the IPC stuff accordingly)